docs/competency-api/graphql.md

```diff
@@ -4,4 +4,487 @@ title: GraphQL
 nav_order: 7
 parent: Competency API
 ---
-# Graphql
\ No newline at end of file
+# GraphQL
+
+## Table of contents
+{: .no_toc .text-delta }
+1. TOC
+{:toc}
+
+## Introduction
+The Competency API uses GraphQL to query and retrieve data. [GraphQL](graphql.org) is a query language for APIs that gives clients control over what data is asked for.
+
+To utilize GraphQL in our API, send a GET request to the [GraphQL route](#definition) with a query document attached to the body. The query document tells the API what data should be returned. The data that is returned is read-only, and cannot be used to manipulate data in the server.
```

Comment:
```suggestion
To utilize GraphQL in our API, send a POST request to the [GraphQL route](#definition) with a query document attached to the body. The query document tells the API what data should be returned. The data that is returned is read-only, and cannot be used to manipulate data in the server.
```

Sending a GET request would require putting the query in the url as a query string, since GET requests do not have request bodys